fitwsarah_frontend/src/App.js

```diff
@@ -1,10 +1,18 @@
-import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import FitnessServiceList from './views/HomePage/FitnessService';
+import Profile from './views/ProfilePage/Account';
+import Settings from './components/clientProfile/setting';
 
 function App() {
-  return (
-    <FitnessServiceList></FitnessServiceList>
-  );
+    return (
+        <Router>
+            <Routes>
+                <Route path="/" element={<FitnessServiceList />} />
+                <Route path="/profile" element={<Profile />} />
+                <Route path="/settings" element={<Settings />} />
+            </Routes>
+        </Router>
+    );
```

Comment:
Can you explain what route does?

Reply:
Absolutely, this route allows it so in the nav logged.js file, when the profile page or settings page is called it makes sure that when its clicked n it renders the page.